app/views/playback/status/TrustClaimedView.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.routes._
+@import models.Mode
+
+@this(
+main_template: MainTemplate
+)
+
+@(utr: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+title = messages("trustClaimed.title")
+) {
+
+<span class="govuk-caption-xl">@messages("trustClaimed.subheading", utr)</span>
+
+@components.heading("trustClaimed.heading")
+
+<p>@messages("trustClaimed.p1")</p>
+
+<p>@messages("trustClaimed.p2") <a id="help-link" href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/trusts" target="_blank">@messages("trustClaimed.p2.a")</a></p>
```

Comment:
"Sign out of this account" CTA button missing?

Reply:
Yeah this relates to TRUS-1992 which hasn't been implemented yet